guides/source/active_support_core_extensions.md

```diff
@@ -2889,9 +2889,9 @@ As the example depicts, the `:db` format generates a `BETWEEN` SQL clause. That
 
 NOTE: Defined in `active_support/core_ext/range/conversions.rb`.
 
-### `include?`
+### `===`, `include?`, and `cover?`
 
-The methods `Range#include?` and `Range#===` say whether some value falls between the ends of a given instance:
+The methods `Range#===`, `Range#include?` and `Range#cover?` say whether some value falls between the ends of a given instance:
```

Comment:
http://guides.rubyonrails.org/v5.2.0/api_documentation_guidelines.html#oxford-comma

> Please use the Oxford comma ("red, white, and blue", instead of "red, white and blue").

🤓

Reply:
Oxford in a comma, it's serious ;)